assets/js/script.js

```diff
@@ -0,0 +1,137 @@
+/*  ------=========     Menu    =========------   */
+
+function mobileMenu() {
+  document.querySelector('HEADER').classList.toggle('header-menu');
+  document.querySelector('.headerLogo').classList.toggle('hide');
+  document.querySelector('body').classList.toggle('overFlow');
+}
+
+document.querySelector('#menuIcon').addEventListener('click', () => {
+  mobileMenu();
+});
+
+/*    ------=========     Data    =========------    */
+
+const projectData = [
```

Comment:
The section "Featured speakers" is generated dynamically with JavaScript 🔥 Superb 🚀